pages/wine-cellar-page.js

```diff
@@ -117,28 +119,22 @@ const NavBar = styled.footer`
   gap: 30px;
   background-color: rgba(31, 31, 35, 1);
 `;
+
 const StyledQuizButton = styled.button`
   font-family: Georgia, 'Times New Roman', Times, serif;
   font-size: 42px;
   padding-bottom: 9px;
   color: #8a98a5;
   border: none;
   background-color: rgba(31, 31, 35, 1);
-  :hover {
-    border-bottom: 3px solid white;
-    margin-top: 1px;
-  }
 `;
+
 const StyledWineGlasButton = styled.button`
```

Comment:
This is a minor nitpick, but try to check your spelling occasionally. It might create some difficulties down the line, if other people don't know something is supposed to be misspelled. ; )